tests/mvp-demo/mvp-demo.go

```diff
@@ -271,7 +272,7 @@ var _ = framework.MvpDemoSuiteDescribe("MVP Demo tests", Label("mvp-demo"), func
 					return false
 				}
 				return pipelineRun.HasStarted()
-			}, pipelineRunStartedTimeout, defaultPollingInterval).Should(BeTrue())
+			}, pipelineRunStartedTimeout, defaultPollingInterval).Should(BeTrue(), fmt.Sprintf("component %q did not start a pipelinerun named %q in namespace %q from release %q in namespace %q in time", componentName, pipelineRun.Name, managedNamespace, release.Name, release.Namespace))
```

Comment:
IIUC, I'm seeing a `runtime error: invalid memory address or nil pointer dereference` error on this line in the [build-log.txt of the latest CI run](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-appstudio_e2e-tests/523/pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e/1658941664380588032#1:build-log.txt%3A1075).

Is it occurring when the pipelineRun is `nil` and we're calling `pipelineRun.name` here?

Reply:
yep good catch @dheerajodha 

```
   Full Stack Trace
    github.com/redhat-appstudio/e2e-tests/tests/mvp-demo.glob..func1.3.7()
    	/tmp/tmp.OJpe8peXxe/tests/mvp-demo/mvp-demo.go:275 +0x1ac
```

I will fix and push an update

Reply:
update pushed

Reply:
Thank you!